src/PythonProjectBootstrapper/package/hooks/post_gen_project.py

```diff
@@ -160,6 +155,12 @@
 }
 
 
+# ----------------------------------------------------------------------
+def UpdateBootstrapExecutionPermissions():
+    status = os.stat("./Bootstrap.sh")

```

Comment:
Path has a [stat method](https://docs.python.org/3/library/pathlib.html#pathlib.Path.stat) that should be used instead of `os.stat`.

Reply:
Done